site/content/events/2015-siliconvalley/_sponsors.txt

```diff
@@ -11,16 +11,20 @@ end
 %>
 
 <% @gsponsors = [
- { :image => "jut.png", :name => "Jut", :link => "http://www.jut.io", :width => '100px', :height => '100px' },
- { :image => "docker.png", :name => "Jut", :link => "http://www.docker.io", :width => '100px', :height => '100px' },
  { :image => 'appformix.png', :name => 'AppFormix', :link => 'http://www.appformix.com', :width => '150px', :height => '50px' },
- { :image => 'sysdig.png', :name => 'Sysdig', :link => 'http://www.sysdig.com', :width => '100px', :height => '100px' },
+ { :image => 'chef.png', :name => 'Chef', :link => 'http://www.chef.io', :width => '100px', :height => '100px' },
  { :image => 'cloudpassage.png', :name => 'CloudPassage', :link => 'http://www.cloudpassage.com', :width => '100px', :height => '100px' },
- { :image => 'neptune.png', :name => 'Neptune', :link => 'http://www.neptune.io', :width => '100px', :height => '100px' },
+ { :image => "datadog.png", :name => "Datadog", :link => "http://www.datadoghq.com", :width => '100px', :height => '100px' },
  { :image => 'delphix.png', :name => 'Delphix', :link => 'http://www.delphix.com', :width => '100px', :height => '100px' },
- { :image => 'chef.png', :name => 'Chef', :link => 'http://www.chef.io', :width => '100px', :height => '100px' },
- { :image => 'signalfx.jpg', :name => 'SignalFx', :link => 'https://signalfx.com/', :width => '100px', :height => '100px' },
+ { :image => "docker.png", :name => "Jut", :link => "http://www.docker.io", :width => '100px', :height => '100px' },
```

Comment:
Are you sure you want to list Docker as Jut? I think this might be a copypasta error.